frontend/src/features/map_planning/routes/MapWrapper.tsx

```diff
@@ -63,6 +64,25 @@ function usePlantLayer({ mapId, layerId }: UseLayerParams) {
   return query;
 }
 
+/**
+ * Hook that initializes the base layer by fetching it and adding it to the store.
+ */
+function useBaseLayer({ mapId, layerId, enabled }: UseLayerParams) {
+  const query = useQuery({
+    queryKey: ['baselayer', mapId, layerId],
+    queryFn: () => getBaseLayerImage(mapId, layerId),
+    enabled,
```

Comment:
```suggestion
    queryKey: ['baselayer', mapId, layerId],
    queryFn: () => getBaseLayerImage(mapId, layerId),
    refetchOnWindowFocus: false,
    enabled,
```

We have to see if this needs to be changed like it is done in the plant layer.
`cacheTime: 0`,
`staleTime: Infinity`
The image config should ideally only be refetched if mapId or layerId changes.

Reply:
Thank you for your feedback :+1:.